src/components/footer/Footer.jsx

```diff
@@ -1,35 +1,79 @@
 import React from 'react';
 import classes from './Footer.module.scss';
-import { privacyNotice } from '../../assets';
+// import { nowUOrange } from '../../assets';
+import { privacyNotice, nowUOrange } from '../../assets';
+import { AppDownloadButtons } from '../../components';
 
 const Footer = () => {
   return (
-    <div className={classes.footer}>
-      <div className={classes.links}>
-        <a href={privacyNotice} target="_blank" rel="noopener noreferrer">
-          Privacy Policy
-        </a>
+    <div className={classes.footerBackground}>
+      <div className={classes.logo}>
+        <img src={nowUOrange} alt="logo" />
       </div>
-      <div className={classes.socialLinks}>
-        <a
-          href="https://www.facebook.com/nowufb"
-          className="fa fa-facebook"
-          target="_blank"
-          rel="noopener noreferrer"
-        ></a>
-        <a href="https://twitter.com/now_u_app" className="fa fa-twitter" target="_blank" rel="noopener noreferrer"></a>
-        <a
-          href="https://www.instagram.com/now_u_app/"
-          className="fa fa-instagram"
-          target="_blank"
-          rel="noopener noreferrer"
-        ></a>
-        <a
-          href="https://www.linkedin.com/company/now-u/?viewAsMember=true"
-          className="fa fa-linkedin"
-          target="_blank"
-          rel="noopener noreferrer"
-        ></a>
+      <div className={classes.footer}>
+        <div className={classes.footerLeft}>
+          <div className={classes.linksColumnOne}>
+            <div className={classes.links}>
+              <a href="/campaigns" className={classes.links}>Campaigns</a>
+            </div>
+            <div className={classes.links}>
+              <a href="/now-u-app" className={classes.links}>Now-u app</a>
+            </div>
+            <div className={classes.links}>
+              <a href="aboutus" className={classes.links}>About us</a>
+            </div>
+            <div className={classes.links}>
+              <a href="get-in-touch" className={classes.links}>Get in touch</a>
+            </div>
+            <div className={classes.links}>
+              <a href="press" className={classes.links}>Press</a>
+            </div>
+          </div>
+          <div className={classes.linksColumnTwo}>
+            <div className={classes.links}>
+              <a href="faqs" className={classes.links}>FAQ</a>
+            </div>
+            <div className={classes.links}>
+              <a href="/" className={classes.links}>Blog</a>
+            </div>
+            <div className={classes.links}>
+              <a href="/" className={classes.links}>Newsletter</a>
```

Comment:
This should trigger the newsletter pop up - not sure how we want to do this tho? What should we show if we know the user has already signed up? Might not be a this pr issue tho, so feel free to create a new issue for this

Reply:
Perhaps a 'Youve already signed up!" message - Maybe we can speak to the design team about this? And perhaps also an option to resend/resignup to the newsletter if they think the first signup didnt work, like ' didnt get your last newsletter? resend newsletter'